batch/src/test/scala/uk/gov/ons/addressindex/models/HybridAddressEsDocumentSpec.scala

```diff
@@ -363,7 +436,7 @@ class HybridAddressEsDocumentSpec extends WordSpec with Matchers {
         expectedNag("townName").toString, expectedNag("postcodeLocator").toString)
 
       // Then
-      result shouldBe "Acme Stats PLC, 6473FF-6623JJ, The Building Name, A Training Centre, 56HH-7755OP And Another Street Descriptor, Locality Xyz, Town B, KL8 7HQ"
```

Comment:
Why was "The" removed? Was it to get a test to work? Or just because it's unloved?

Reply:
I think it used to get treated as a lower case part and get changed to the and fail the test, I think it should work correctly if I add 'The' again.